manifests/o/octobuild/octobuild/0.4.0/octobuild.octobuild.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: octobuild.octobuild
+PackageVersion: 0.4.0
+PackageLocale: en-US
+Publisher: Marat Radchenko

```

Comment:
```suggestion
Publisher: Artem V. Navrotskiy
```

Reply:
I'm not sure what is ARP, but okay.

Reply:
> I'm not sure what is ARP, but okay.

My apologies, I should've been more clear on that. ARP stands for Add or Remove Program, the list of installed programs. The easiest way to get to this menu is via Run -> `appwiz.cpl` or open up the control panel and click on "Uninstall a program".

![image](https://user-images.githubusercontent.com/52826317/215535387-094f5c11-a60b-4a60-b79d-4773aa117783.png)
(It's in Norwegian, Navn = Name, Utgiver = Publisher).

The regedit string for this this specific application is located here:
`HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\{BCA869D2-6FDD-4896-B1A2-C9F47132BED0}`

![image](https://user-images.githubusercontent.com/52826317/215535230-b0e74dd5-2a6e-44eb-a9c5-fef5c4686304.png)

Reply:
Oh, okay.